app/service/rpc_service.ts

```diff
@@ -24,14 +24,20 @@ export type BuildBuddyServiceRpcName = RpcMethodNames<buildbuddy.service.BuildBu
 class RpcService {
   service: ExtendedBuildBuddyService;
   events: Subject<string>;
-  requestContext = new context.RequestContext({
-    timezoneOffsetMinutes: new Date().getTimezoneOffset(),
```

Comment:
nit: we don't need to support IE11/Opera Mini etc. if you want to call `Intl.DateTimeFormat().resolvedOptions().timeZone` here without any conditionals etc https://caniuse.com/?search=resolvedOptions

Reply:
Fiiine